WordPress/Classes/ViewRelated/Media/Tenor/TenorMediaGroup.swift

```diff
@@ -0,0 +1,27 @@
+import WPMediaPicker
+
+final class TenorMediaGroup: NSObject, WPMediaGroup {
+    func name() -> String {
+        return String.tenor
+    }
+
+    func image(with size: CGSize, completionHandler: @escaping WPMediaImageBlock) -> WPMediaRequestID {
+        return 0
+    }
+
+    func cancelImageRequest(_ requestID: WPMediaRequestID) {
+        //
+    }
+
+    func baseGroup() -> Any {
+        return ""
+    }
+
+    func identifier() -> String {
+        return "group id"
+    }
+
+    func numberOfAssets(of mediaType: WPMediaType, completionHandler: WPMediaCountBlock? = nil) -> Int {
+        return 10
```

Comment:
The `TenorMediaGroup` is created just to make the `TenorDataSource` conformed to the `WPMediaCollectionDataSource` protocol. Since we haven't supported media groups (e.g group by categories) in all media sources (StockPhotos, Giphy, Tenor). So 10 or any doesn't matter. StockPhotos uses 10, Giphy uses 10, Tenor (is a Giphy clone), has 10 too. :d